app/views/works/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<header class="details-page">
+  <h2><%= @work.title %></h2>
+    <section class="details">
+      Created by: <%= @work.creator %><br>
+      Published: <%= @work.pub_yr %><br>
+      <%= @work.description %><br>
+    </section>
+</header>
+  
+  <section class="actions">
+    <%= link_to "Back to media ranks", root_path, class: "btn btn-primary btn-lg" %>
+    <%= link_to "Edit", edit_work_path(@work.id), class: "btn btn-primary btn-lg" %>
+    <%= link_to "Upvote", class: "btn btn-primary btn-lg" %>
+    <%= link_to "Delete", class: "btn btn-danger btn-lg" %> 
+  </section>
+</header>
+
+<h3><%= @work.votes.count %> Votes for this <%= @work.category%></h3>
+
+<table class ="votes">
+ <thead>
+  <th>User</th>
+  <th>Date</th>
+  </thead>
+  <tbody>
+    <% @work.votes.each do |vote| %>
+    <tr>
+      <td><%= link_to vote.user, user_path(vote.user) %></td>
+      <td> <%= vote.date %> </td>
+    </tr>
```

Comment:
This table doesn't quite render the way you probably intended. Something like

```html+erb
<tr>
  <td><%= link_to vote.user.name, user_path(vote.user) %></td>
  <td><%= vote.created_at %></td>
</tr>
```

might be closer to what you want.